db/warehouse/migrate/20231203221840_idx_warehouse_versions.rb

```diff
@@ -0,0 +1,10 @@
+class IdxWarehouseVersions < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    # indexes for audit reports
+    add_index :versions, :client_id, algorithm: :concurrently
+    add_index :versions, :enrollment_id, algorithm: :concurrently
+    add_index :versions, :project_id, algorithm: :concurrently
```

Comment:
We should make sure and watch this on staging/QA deploys.  I've seen weird behavior with concurrent index adding (as in sometimes it hangs forever, which is what it's trying to avoid.)